Tuto-GUDHI-DTM-filtrations.ipynb

```diff
@@ -116,7 +116,7 @@
    "outputs": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    gudhi.plot_persistence_diagram(diagram_DTM, legend=True)
</pre><p>Shouldn't we make <code>legend=True</code> the default in Gudhi? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>https://github.com/GUDHI/gudhi-devel/issues/732</p>